.travis.yml

```diff
@@ -47,6 +60,11 @@ deploy:
   script: bash travis-docker-push.sh
   on:
     all_branches: true
+- provider: script
+  script: bash deploy-terraform.sh
+  on:
+    all_branches: true
+    condition: $TRAVIS_BRANCH =~ ^(develop|master)$
```

Comment:
I think this is safer 👍 Easy for us to add another environment here in the rare case we'd need to, for example, `|pentest` or `|testing`.